frontend/i18next.scanner/en/translation.json

```
@@ -348,5 +348,22 @@
   "Maximum size: {{size}} MB": "Maximum size: {{size}} MB",
   "Registration date:": "Registration date:",
   "Cover image of {{publicName}}": "Cover image of {{publicName}}",
-  "Change cover": "Change cover"
+  "Change cover": "Change cover", 
+  "Comment_sub": "Comment", 
```

maybe "noun" is more clear

```suggestion
  "Comment_noun": "Comment", 
```